docs/book/install-and-build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
+cd "${KUBE_ROOT}" || exit 1
+
+os=$(go env GOOS)
+arch=$(go env GOARCH)
+
+# translate arch to rust's conventions (if we can)
+if [[ ${arch} == "amd64" ]]; then
+    arch="x86_64"
+elif [[ ${arch} == "x86" ]]; then
+    arch="i686"
+fi
+
+# translate os to rust's conventions (if we can)
+ext="tar.gz"
+cmd="tar -C /tmp -xzvf"
+case ${os} in
+    windows)
+        target="pc-windows-msvc"
+        ext="zip"
+        cmd="unzip -d /tmp"
+        ;;
+    darwin)
+        target="apple-darwin"
+        ;;
+    linux)
+        # works for linux, too
+        target="unknown-${os}-gnu"
+        ;;
+    *)
+        target="unknown-${os}"
+        ;;
+esac
+
+# grab mdbook
+# we hardcode linux/amd64 since rust uses a different naming scheme
+echo "downloading mdBook-v0.3.1-${arch}-${target}.${ext}"
```

Comment:
should we make the version a variable as well to make it easy when upgrade?

Reply:
I can do that once we have the CI job in place to test to make sure I'm not breaking anything, for now copied this whole file from known working https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/install-and-build.sh